src/oss-metadata/MetadataTool.cs

```diff
@@ -54,7 +54,7 @@ public MetadataTool(IHttpClientFactory httpClientFactory) : base(httpClientFacto
                 BaseProjectManager? projectManager = ProjectManagerFactory.CreateProjectManager(purl, httpClientFactory);
                 if (projectManager != null)
                 {
-                    metadata = await projectManager.GetPackageMetadata(purl);
+                    metadata = await projectManager.GetPackageMetadata(purl, false);
```

Comment:
This change default behavior from using cache to not using cache.  Perhaps this tool needs a use cache argument to specify?